src/main/java/com/digitalasset/testing/ledger/DamlScriptRunner.java

```diff
@@ -101,7 +91,7 @@ public DamlScriptRunner build() {
     private ProcessBuilder command() {
       String sandboxHost = "localhost";
       return new ProcessBuilder()
-          .directory(projectRoot.toFile())
+          .directory(damlRoot.toFile())
```

Comment:
Careful, damlRoot now can be null. ProcessBuilder accepts `null` values, but this here will throw NPE. I'd consider using Optional

Reply:
No, it cannot be. It is an _internal_ state, and `com.digitalasset.testing.ledger.DamlScriptRunner.Builder#build` is ensures that it actually points to a folder with a `daml.yaml` in it.

Side note: even it was `null`, I'd keep it, because that would mean an incorrect setup of the runner.